### PR TITLE
Making cross origin objects.sub.html work in restricted test environments

### PR DESCRIPTION
**Pull request by youennf**

### CODE REVIEW

**Comment by youennf** [comment]
@zcorpan @cdumez

File: html/browsers/origin/cross-origin-objects/cross-origin-objects.html [hunk 1]
```diff
@@ -51,10 +53,9 @@
  */
 
 addTest(function() {
-  // Note: we do not check location.hostname as its default port semantics are hard to reflect
-  //       statically
-  assert_equals(location.hostname, '{{domains[]}}', 'Need to run the top-level test from domain {{domains[]}}');
-  assert_equals(location.port, '{{location[port]}}', 'Need to run the top-level test from port {{location[port]}}')
+  // Note: we do not check location.hostname as its default port semantics are hard to reflect statically
```
Comment (zcorpan) [inline html/browsers/origin/cross-origin-objects/cross-origin-objects.html]: While here, can you change this to location.host?

**Comment by zcorpan** [comment]
Looks OK to me, but maybe @bholley also wants to check this?

**Comment by bholley** [comment]
Seems fine at a quick glance - IIUC none of the invariants are really changing, right?

**Comment by youennf** [comment]
What is tested should not be changed, we just split the test in two and update the cross-origin URL computation for one of the test to also be run in environments that can only use 127.0.0.1/localhost

**Pull request merged**